docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1242,10 +1237,10 @@ importance] values per document. By default, it is zero and no feature importanc
 calculation occurs.
 end::inference-config-regression-num-top-feature-importance-values[]
 
-tag::inference-config-regression-results-field[]
-Specifies the field to which the inference prediction is written. Defaults to
-`predicted_value`.
-end::inference-config-regression-results-field[]
+tag::inference-config-results-field[]
+The field that is added to incoming documents to contain the inference
```

Comment:
FYI I merged the two results_field definitions per discussions related to https://github.com/elastic/elasticsearch/pull/54421